.github/labels.yml

```diff
@@ -55,18 +61,27 @@
 - name: "Type: 🧹 chore"
   description: "Routine tasks, maintenance, or non-user-facing changes."
   color: "FAD8C7"
+- name: "Type: ❓ question"
+  description: "Not an classical issue. Customer's question."
+  color: "C7FAD8"
```

Comment:
I do not consider it necessary to have this label.

Discussions on issues should be in the `Discussions` section, not in Issues. 

- [ ] should be described in the community guideline.

Reply:
Ok, it may be lagacy tybe because it was created before Github's discussions released.

I will drop it.